.github/workflows/build-binary.yaml

```diff
@@ -24,32 +24,38 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          pip install -r requirements.txt
-          pip install imageio
+          pip install -Ur requirements.txt
+          pip install -U pillow
+          pip install -U pyinstaller
```

Comment:
should these be specific versions rather than the latest?

Reply:
Done